conf/messages

```diff
@@ -111,6 +111,8 @@ emailPreference.title = Do you want us to send you emails instead of letters?
 emailPreference.hint = We currently send messages about your VAT by letter.
 emailPreference.error = Select yes if you want communications by email
 
+LetterPreference.error = Select yes if you want communications by letter
```

Comment:
A bit picky, but could you change "LetterPreference.error" to "letterPreference.error" here and in the welsh messages